src/retabled/core.cljc

```diff
@@ -177,7 +182,12 @@
                                   :no-paging
                                   nil)
             entries (curate-entries paging-controls entries SORT FILTER)]      
-        [:table.table
+        [:table.table (when (:table-scroll-bar? controls)
+                        {:style {"height" "450px"
```

Comment:
these default styles should be responsive units (not px), and need to be easily over-written

Reply:
actually, this is good. If they want to they can just style everything themselves, so we are justified in in-lining here

Reply:
having defaults is good, but the units need to be made responsive